backend/src/test/java/gov/cdc/usds/simplereport/test_util/TestDataBuilder.java

```diff
@@ -67,8 +68,7 @@ public static Person createEmptyPerson(boolean withOrganizationObject) {
 
   public static Facility createEmptyFacility(boolean includeValidDevice) {
     DeviceType device = includeValidDevice ? createEmptyDeviceWithLoinc() : null;
-    return new Facility(
-        null, null, null, null, null, null, null, device, null, Collections.emptyList());
+    return new Facility(FacilityBuilder.builder().defaultDeviceType(device).build());
```

Comment:
does this need to have `configuredDevices` set to an empty list? this could be why the backend tests are failing

Reply:
oh thanks for catching Merethe! fixed.